{
  "info": {
    "author": "Google LLC",
    "author_email": "googleapis-packages@google.com",
    "bugtrack_url": null,
    "classifiers": [
      "Development Status :: 4 - Beta",
      "Intended Audience :: Developers",
      "License :: OSI Approved :: Apache Software License",
      "Operating System :: OS Independent",
      "Programming Language :: Python",
      "Programming Language :: Python :: 3",
      "Programming Language :: Python :: 3.10",
      "Programming Language :: Python :: 3.6",
      "Programming Language :: Python :: 3.7",
      "Programming Language :: Python :: 3.8",
      "Programming Language :: Python :: 3.9",
      "Topic :: Internet"
    ],
    "description": "Python Client for Cloud Error Reporting\n=============================================\n\n|beta| |pypi| |versions|\n\nThe Cloud `Error Reporting`_ API counts, analyzes and aggregates the\ncrashes in your running cloud services.  A centralized error management\ninterface displays the results with sorting and filtering capabilities. A\ndedicated view shows the error details: time chart, occurrences, affected user\ncount, first and last seen dates and a cleaned exception stack trace. Opt-in\nto receive email and mobile alerts on new errors.\n\n- `Client Library Documentation`_\n- `Product Documentation`_\n\n.. _Error Reporting: https://cloud.google.com/error-reporting/\n.. _Client Library Documentation: https://cloud.google.com/python/docs/reference/clouderrorreporting/latest\n.. _Product Documentation: https://cloud.google.com/error-reporting/reference/\n.. |beta| image:: https://img.shields.io/badge/support-beta-orange.svg\n   :target: https://github.com/googleapis/google-cloud-python/blob/main/README.rst#beta-support\n.. |pypi| image:: https://img.shields.io/pypi/v/google-cloud-error-reporting.svg\n   :target: https://pypi.org/project/google-cloud-error-reporting/\n.. |versions| image:: https://img.shields.io/pypi/pyversions/google-cloud-error-reporting.svg\n   :target: https://pypi.org/project/google-cloud-error-reporting/\n\nQuick Start\n-----------\n\nIn order to use this library, you first need to go through the following steps:\n\n1. `Select or create a Cloud Platform project.`_\n2. `Enable billing for your project.`_\n3. `Enable the Google Cloud Error Reporting API.`_\n4. `Setup Authentication.`_\n\n.. _Select or create a Cloud Platform project.: https://console.cloud.google.com/project\n.. _Enable billing for your project.: https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project\n.. _Enable the Google Cloud Error Reporting API.:  https://cloud.google.com/error-reporting\n.. _Setup Authentication.: https://googleapis.dev/python/google-api-core/latest/auth.html\n\nInstallation\n~~~~~~~~~~~~\n\nInstall this library in a `virtualenv`_ using pip. `virtualenv`_ is a tool to\ncreate isolated Python environments. The basic problem it addresses is one of\ndependencies and versions, and indirectly permissions.\n\nWith `virtualenv`_, it's possible to install this library without needing system\ninstall permissions, and without clashing with the installed system\ndependencies.\n\n.. _`virtualenv`: https://virtualenv.pypa.io/en/latest/\n\n\nSupported Python Versions\n^^^^^^^^^^^^^^^^^^^^^^^^^\nPython >= 3.6\n\nDeprecated Python Versions\n^^^^^^^^^^^^^^^^^^^^^^^^^^\nPython == 2.7.\n\nThe last version of this library compatible with Python 2.7 is google-cloud-error-reporting==0.34.0.\n\n\nMac/Linux\n^^^^^^^^^\n\n.. code-block:: console\n\n    pip install virtualenv\n    virtualenv <your-env>\n    source <your-env>/bin/activate\n    <your-env>/bin/pip install google-cloud-error-reporting\n\n\nWindows\n^^^^^^^\n\n.. code-block:: console\n\n    pip install virtualenv\n    virtualenv <your-env>\n    <your-env>\\Scripts\\activate\n    <your-env>\\Scripts\\pip.exe install google-cloud-error-reporting\n\n\n\nNext Steps\n~~~~~~~~~~\n\n-  Read the `Client Library Documentation`_ for Google Cloud Error Reporting\n   API to see other available methods on the client.\n-  Read the `Product documentation`_ to learn\n   more about the product and see How-to Guides.\n\n\n",
    "description_content_type": "",
    "docs_url": null,
    "download_url": "",
    "downloads": {
      "last_day": -1,
      "last_month": -1,
      "last_week": -1
    },
    "home_page": "https://github.com/googleapis/python-error-reporting",
    "keywords": "",
    "license": "Apache 2.0",
    "maintainer": "",
    "maintainer_email": "",
    "name": "google-cloud-error-reporting-tcspowerflow",
    "package_url": "https://pypi.org/project/google-cloud-error-reporting-tcspowerflow/",
    "platform": "Posix; MacOS X; Windows",
    "project_url": "https://pypi.org/project/google-cloud-error-reporting-tcspowerflow/",
    "project_urls": {
      "Homepage": "https://github.com/googleapis/python-error-reporting"
    },
    "release_url": "https://pypi.org/project/google-cloud-error-reporting-tcspowerflow/1.4.1/",
    "requires_dist": [
      "google-cloud-logging (<3.0.0dev,>=1.14.0)",
      "google-api-core[grpc] (<3.0.0dev,>=1.28.0)",
      "proto-plus (>=1.4.0)"
    ],
    "requires_python": ">=3.6",
    "summary": "Error Reporting API client library",
    "version": "1.4.1",
    "yanked": false,
    "yanked_reason": null
  },
  "last_serial": 12564856,
  "releases": {
    "1.4.1": [
      {
        "comment_text": "",
        "digests": {
          "blake2b_256": "e2aeb381bd21935589590d629d83c9b75f4bec9cd6fa631f1834d9397f323918",
          "md5": "7bfd8cf18d80164b727606f73b47e508",
          "sha256": "d4bf2ef27302fbdef0e411bdeab26708ea586bb82fa1b454f97920f290c6cf29"
        },
        "downloads": -1,
        "filename": "google_cloud_error_reporting_tcspowerflow-1.4.1-py2.py3-none-any.whl",
        "has_sig": false,
        "md5_digest": "7bfd8cf18d80164b727606f73b47e508",
        "packagetype": "bdist_wheel",
        "python_version": "py2.py3",
        "requires_python": ">=3.6",
        "size": 99247,
        "upload_time": "2022-01-13T19:02:54",
        "upload_time_iso_8601": "2022-01-13T19:02:54.551990Z",
        "url": "https://files.pythonhosted.org/packages/e2/ae/b381bd21935589590d629d83c9b75f4bec9cd6fa631f1834d9397f323918/google_cloud_error_reporting_tcspowerflow-1.4.1-py2.py3-none-any.whl",
        "yanked": false,
        "yanked_reason": null
      },
      {
        "comment_text": "",
        "digests": {
          "blake2b_256": "5311dfbd204b90e69a25a9b01c352be34286faa8398aabfd08940de68afd62a3",
          "md5": "f82ed06eb42ce8bdaee5851cfa719564",
          "sha256": "3e35b2e13f67d88fa6775e8471d25aa852f227c3dfb230b1324012602e0ebdfb"
        },
        "downloads": -1,
        "filename": "google-cloud-error-reporting-tcspowerflow-1.4.1.tar.gz",
        "has_sig": false,
        "md5_digest": "f82ed06eb42ce8bdaee5851cfa719564",
        "packagetype": "sdist",
        "python_version": "source",
        "requires_python": ">=3.6",
        "size": 77689,
        "upload_time": "2022-01-13T19:02:56",
        "upload_time_iso_8601": "2022-01-13T19:02:56.411470Z",
        "url": "https://files.pythonhosted.org/packages/53/11/dfbd204b90e69a25a9b01c352be34286faa8398aabfd08940de68afd62a3/google-cloud-error-reporting-tcspowerflow-1.4.1.tar.gz",
        "yanked": false,
        "yanked_reason": null
      }
    ]
  },
  "urls": [
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "e2aeb381bd21935589590d629d83c9b75f4bec9cd6fa631f1834d9397f323918",
        "md5": "7bfd8cf18d80164b727606f73b47e508",
        "sha256": "d4bf2ef27302fbdef0e411bdeab26708ea586bb82fa1b454f97920f290c6cf29"
      },
      "downloads": -1,
      "filename": "google_cloud_error_reporting_tcspowerflow-1.4.1-py2.py3-none-any.whl",
      "has_sig": false,
      "md5_digest": "7bfd8cf18d80164b727606f73b47e508",
      "packagetype": "bdist_wheel",
      "python_version": "py2.py3",
      "requires_python": ">=3.6",
      "size": 99247,
      "upload_time": "2022-01-13T19:02:54",
      "upload_time_iso_8601": "2022-01-13T19:02:54.551990Z",
      "url": "https://files.pythonhosted.org/packages/e2/ae/b381bd21935589590d629d83c9b75f4bec9cd6fa631f1834d9397f323918/google_cloud_error_reporting_tcspowerflow-1.4.1-py2.py3-none-any.whl",
      "yanked": false,
      "yanked_reason": null
    },
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "5311dfbd204b90e69a25a9b01c352be34286faa8398aabfd08940de68afd62a3",
        "md5": "f82ed06eb42ce8bdaee5851cfa719564",
        "sha256": "3e35b2e13f67d88fa6775e8471d25aa852f227c3dfb230b1324012602e0ebdfb"
      },
      "downloads": -1,
      "filename": "google-cloud-error-reporting-tcspowerflow-1.4.1.tar.gz",
      "has_sig": false,
      "md5_digest": "f82ed06eb42ce8bdaee5851cfa719564",
      "packagetype": "sdist",
      "python_version": "source",
      "requires_python": ">=3.6",
      "size": 77689,
      "upload_time": "2022-01-13T19:02:56",
      "upload_time_iso_8601": "2022-01-13T19:02:56.411470Z",
      "url": "https://files.pythonhosted.org/packages/53/11/dfbd204b90e69a25a9b01c352be34286faa8398aabfd08940de68afd62a3/google-cloud-error-reporting-tcspowerflow-1.4.1.tar.gz",
      "yanked": false,
      "yanked_reason": null
    }
  ],
  "vulnerabilities": []
}